lib/activerecord-import/import.rb

```diff
@@ -275,7 +275,11 @@ def import_helper( *args )
           # this next line breaks sqlite.so with a segmentation fault
           # if model.new_record? || options[:on_duplicate_key_update]
             column_names.map do |name|
-              model.read_attribute_before_type_cast(name.to_s)
+              if model.class.column_defaults[name.to_s].is_a? Integer
+                model.read_attribute(name.to_s)
+              else
+                model.read_attribute_before_type_cast(name.to_s)
```

Comment:
`read_attribute ` is slightly slower than `read_attribute_before_type_cast ` for type casting. 

But for rails enum, type casting happens **after** enum value checking. For example, `Person.new(age: "10")` is fine even if age column is an integer, but `Person.new(status: "1")` will raise `ArgumentError: '1' is not a valid status` given status is an enum integer attribute.

One thing I just discovered, `model.class.column_defaults` is very slow. So I am going to make another commit to cache it or something.